backend/dataall/db/api/environment.py

```diff
@@ -352,10 +352,6 @@ def remove_group(session, username, groups, uri, data=None, check_perm=None):
                 models.RedshiftCluster.environmentUri
```

Comment:
`pipelinesEnabled` should be migrated to parameters

Reply:
Also don't forget to delete it from `backend/dataall/db/models/Environment.py` and add it to the migration script `backend/migrations/versions/5fc49baecea4_add_enviromental_parameters.py`

Reply:
And in `backend/dataall/modules/datapipelines/services/datapipelines_service.py` Line43 and 117

Reply:
I moved it to env params. Thanks for the hints!